jac-uk/admin
# Targeted outreach fix

## What's included?
- Routing changes so the targeted outreach report page loads under its own route (previously would load an candidate page in error)
- Spelling mistake on outreach report page (oureach)
- Table resize to fit all columns and content for TA report

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- Load the targeted outreach report page, refresh the page. check same page loads rather than blank candidate page
- Enter a valid NI number eg. 'qq123456c'
- observe all columns and data

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
before
<img width="897" alt="image" src="https://user-images.githubusercontent.com/44227249/216661978-6eba061b-3f9b-4328-adc3-645e1f9faf08.png">
after 
<img width="1253" alt="image" src="https://user-images.githubusercontent.com/44227249/216662259-e7c732e4-815a-4e31-b550-0f89c1f2616b.png">

<img width="1223" alt="image" src="https://user-images.githubusercontent.com/44227249/216662175-8dd20133-575e-4d70-a99a-18d66ea18ead.png">

<img width="1248" alt="image" src="https://user-images.githubusercontent.com/44227249/216663160-280591f6-3c6a-4f81-ac40-a21db9411b86.png">

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
